README.md

```diff
@@ -11,3 +11,16 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+Applying the log base change formula to change both logarithms to a base of e:  
+$log_{5} n = \frac{ln(n)}{ln 5} = \frac{1}{ln 5} * ln(n)$  
+
+$log_{2} n = \frac{ln(n)}{ln 2} = \frac{1}{ln 2} * ln(n)$  
+Both logarithms follow the formula of *constant* * ln(n).  
+It follows that, for either of the two logarithms:  
+$T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$  
+In more concrete words:   
+$\frac{ln(n)}{ln 5} \in O(\frac{ln(n)}{ln 2})$  since  $\exists c, n_0: \frac{ln(n)}{ln 5} \leq c \cdot \frac{ln(n)}{ln 2} \forall n \geq n_0$  
```

Comment:
You are assuming a particular $T(n)$ here -- you have to show it for all possible $T(n)$.